src/api/reportApi.ts

```diff
@@ -0,0 +1,34 @@
+// import { AxiosError, AxiosResponse } from "axios";
+import { instance } from "./axiosInstance";
+
+
+export const reportApi = {
+
+  filterDateToReportVisit: async (data: {
+    type: string,
+    startDate: string,
```

Comment:
whether it is possible to take out this attrs to the interface?)

Reply:
yes this is a pertinent remark, I will correct, thank you)